app/app/src/main/java/org/omecproject/up4/impl/Up4DeviceManager.java

```diff
@@ -904,6 +939,20 @@ public UpfCounter readCounter(int counterIdx, DeviceId device) throws UpfProgram
         return upfProgrammable.readCounter(counterIdx);
     }
 
+    @Override
+    public void resetAllSessionMeters() throws UpfProgrammableException {
```

Comment:
m suggestion was more to use the info we have from the pipeconf to do a blind reset without the need to read first